package-lock.json

```diff
@@ -3900,6 +3893,27 @@
       "integrity": "sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=",
       "dev": true
     },
+    "minipass": {
```

Comment:
minipass

Reply:
https://github.com/isaacs/minipass
minipass is a stream implementation and supports `pipe()`